styles/donut-chart.less

```diff
@@ -0,0 +1,13 @@
+@import 'variables';
+
+.donut-ring-pending {
+  stroke: @gh-background-color-yellow;
+}
+
+.donut-ring-succeeded {
```

Comment:
Are the donut charts already supposed work? Currently they all have a single `<circle>` with the class `donut-ring-success`. So it wouldn't match ☝️ above class and it's invisible.

Anyways, I [resized](https://github.com/atom/github/pull/1523/files#diff-6fa7b7b04ab5913e8c481ea8db68f092R43) the chart to fit the smaller space.